py-polars/polars/expr/expr.py

```diff
@@ -10621,6 +10622,15 @@ def dt(self) -> ExprDateTimeNameSpace:
     # Keep the `list` and `str` properties below at the end of the definition of Expr,
     # as to not confuse mypy with the type annotation `str` and `list`
 
+    @property
+    def bitwise(self) -> ExprBitwiseNameSpace:
```

Comment:
We don't have namespaces for operations. Only for data types. I think we should use prefixes and undersfores instead: `bitwise_`